src/unix/fcitx5/BUILD.bazel

```diff
@@ -6,32 +6,64 @@ load(
 
 package(default_visibility = ["//:__subpackages__"])
 
-mozc_cc_library(
-    name = "mozc_connection",
-    srcs = ["mozc_connection.cc"],
-    hdrs = ["mozc_connection.h"],
-    deps = [
-        ":fcitx_key_util",
-        ":surrounding_text_util",
-        "//base:port",
-        "//base:util",
-        "//base:vlog",
-        "//client",
-        "//protocol:commands_cc_proto",
-    ],
+config_setting(
```

Comment:
Example: 
https://github.com/bazelbuild/examples/blob/b51e3bdd468ce8c4a516d7dca993909dcc84af32/configurations/select_on_build_setting/BUILD

You can see how it defines a flag and have a sane default value. (Hint: string_flag & "//conditions:default" ).

Also, "server" as a name is too generic. Better to define a name with "fcitx" in it so it's unique to fcitx (if you use string_flag you will get it for free, since flag is now --//unix/fcitx5:...